beacon_node/beacon_chain/src/canonical_head.rs

```diff
@@ -513,21 +511,13 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
     /// situation can be rectified. We avoid returning an error here so that calling functions
     /// can't abort block import because an error is returned here.
     pub async fn recompute_head_at_slot(self: &Arc<Self>, current_slot: Slot) {
-        let span = info_span!(
```

Comment:
Not sure why this need to be deleted, were you thinking to create the spans in  `spawn_blocking_handle` automatically using the `task_name`?

It doesn't look like it's creating this span though